saleor/graphql/tax/mutations/tax_class_update.py

```diff
@@ -26,14 +38,17 @@ class TaxClassUpdateError(Error):
 class TaxClassUpdateInput(graphene.InputObjectType):
     name = graphene.String(description="Name of the tax class.")
     update_country_rates = NonNullList(
-        CountryRateInput,
+        CountryRateUpdateInput,
         description=(
             "List of country-specific tax rates to create or update for this tax class."
         ),
     )
     remove_country_rates = NonNullList(
         CountryCodeEnum,
-        description="List of country codes for which to remove the tax class rates.",
+        description=(
+            "List of country codes for which to remove the tax class rates. Note: it "
```

Comment:
```suggestion
            "List of country codes for which to remove the tax class rates. Note: It "
```

Just capitalization after colon